pyproject.toml

```diff
@@ -34,7 +34,7 @@ dependencies = [
   "gradio",
   "lightgbm",
   "loguru",
-  "matplotlib<3.8.0",
+  "matplotlib",
```

Comment:
Why do we no longer need to enforce a version of matplotlib?

Reply:
There was a mpl bug that was the cause of this ceiling (https://github.com/drivendataorg/cyfi/pull/104/commits/e379dec3599f705e192220de8740fc52286a6112) but I can't remember the specifics. In general, we only want floors not ceilings for packages. Ceilings are often a temporary workaround and it looked like this one was outdated (indeed as it was given tests pass with later versions)